src/components/Header.vue

```diff
@@ -13,7 +13,7 @@
             <a class="nav-link" href="#"><small>{{ $t("firstNavbar.aboutRoyalLibrary") | capitalize({ onlyFirstLetter: true }) }}</small></a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="#" v-on:click="toggleLanguage">
+            <a class="nav-link" href="#" @click="toggleLanguage">
```

Comment:
Goodie, I missed this in my last review but the linter caught it afterwards but I never got around to fix it but now you did :)